apps/explorer/src/ui/Radio.tsx

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { RadioGroup as HeadlessRadioGroup } from '@headlessui/react';
+import { type ReactNode } from 'react';
+
+import { type ExtractProps } from '~/ui/types';
+
+export type RadioGroupProps = ExtractProps<typeof HeadlessRadioGroup> & {
+    children: ReactNode;
+    ariaLabel: string;
+};
+
+export function RadioGroup({ ariaLabel, children, ...props }: RadioGroupProps) {
+    return (
+        <HeadlessRadioGroup {...props}>
+            <HeadlessRadioGroup.Label className="sr-only">
+                {ariaLabel}
+            </HeadlessRadioGroup.Label>
+            {children}
+        </HeadlessRadioGroup>
+    );
+}
+
+export type RadioOptionProps = ExtractProps<
+    typeof HeadlessRadioGroup.Option
+> & {
+    label?: string;
+    children?: ReactNode;
+};
+
+export function RadioOption({ label, children, ...props }: RadioOptionProps) {
```

Comment:
is https://www.figma.com/file/T06obgYVOUD2JDGXM8QEDX/01-Components-%3A-Shared?node-id=87-202&t=wazKbalsIVZU5nJq-4 the component in Figma? If so, it looks like the text color for selected state should be `text`hero-dark`

Reply:
Thx for catching it, I was using this here https://www.figma.com/file/kSD0qXCIDZA4KQVNepA5hR/02-Explorer?node-id=5165-189404&t=bnTucWJkrktW4qoG-0

Working on https://github.com/MystenLabs/sui/pull/10773 in parallel